### DISCUSSION TITLE
Ignore destructive changes for deltas in Workflows

### DISCUSSION THREAD

**Discussion by jdgirald** [scolladon/sfdx-git-delta#367] [Q&A]
Hi @scolladon,

When getting destructive changes for Workflows, the subtypes are extracted and added to the destructiveChanges.xml file. For example
```
<?xml version="1.0" encoding="UTF-8"?>
<Package xmlns="http://soap.sforce.com/2006/04/metadata">
    <types>
        <members>Opportunity.qualtrics__Qualtrics</members>
        <name>WorkflowOutboundMessage</name>
    </types>
    <types>
        <members>Opportunity.Update account</members>
        <name>WorkflowRule</name>
    </types>
    <types>
        <members>Opportunity.qualtrics__Qualtrics Example</members>
        <name>WorkflowRule</name>
    </types>
    <version>54.0</version>
</Package>
```

Is it possible to ignore subtypes? For example, I want to ignore destructive changes for the `Opportunity.Update account` workflow rule but leave the other components.

I have tried adding different values for the `ignore ` and ` ignore-destructive` options but it seems to only work with file names, which in this case is going to ignore all the individual components.

Thanks

**Comment by scolladon** [comment]
Hi @jdgirald !

You're right, the plugin does not handle `ignore` configuration when dealing with inFile metadata like `Workflow`. It only applies for files.

I think it could be a great enhancement to also us ignore patterns for inFile metadata.

What do you think?

**Reply by jdgirald** [reply]
Hey @scolladon 

It could be a great enhancement, but not sure if it is worth the effort since Salesforce is deprecating workflows.

**Reply by scolladon** [reply]
I agree...
I'll convert that as an issue anyway
And tag it as not scheduled for a fix
Thanks for raising this
